application/classes/Ushahidi/Repository/Config.php

```diff
@@ -49,21 +55,49 @@ public function update(Entity $entity)
 
 		$config = \Kohana::$config->load($group);
 
+		// Intercom count datasources
+		if ($group === 'data-provider') {
+			$intercomData['num_data_sources'] = 0;
+			foreach ($entity->__get('providers') as $key => $value) {
+				$value === 'true' ? $intercomData['num_data_sources']++ : null;
+			}
+		}
+
 		$immutable = $entity->getImmutable();
-		foreach ($entity->getChanged() as $key => $val) {	
-			if (! in_array($key, $immutable)) {				
-				
+		$intercom_data = [];
+		foreach ($entity->getChanged() as $key => $val) {
+
+			// Emit Intercom Update events
+			if ($key === 'description') {
+				$intercom_data['has_description'] = true;
+			}
+
+			if ($key === 'image_header') {
+				$intercom_data['has_logo'] = true;
+			}
+
+			// New User - set their deployment created date
+			if ($key === 'first_login') {
+				$intercom_data['deployment_created_date'] = date("Y-m-d H:i:s");
```

Comment:
This seems likely to be error prone. Could we do something like inject a real 'deployment_created_date' via migrations? ... or we push deployment creation date from the `platform-cloud-interface` codebase instead since it actually knows when deployments are created.